src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -62,7 +62,16 @@
               :title="noResultsText"
             >{{ noResultsText }}</li>
           </slot>
+          <li v-if="isLoading">
```

Comment:
Im'm curious, why not putting this at the top and then using the `v-if-else`?

Reply:
Yeah, good call!

Reply:
Done!

Reply:
Well I meant: 

- if is loading
- else if is empty (it is implied it is not loading)
...

in that order, thus you don't have to do the is not loading for the empty state by reordering the conditions

Reply:
@matallo let me know if this ok for you